### DISCUSSION TITLE
Transform with style={{ x, y }}

### DISCUSSION THREAD

**Discussion by j-o-o** [pmndrs/react-spring#1233] [Support] [answered]
Hello, quick question: I tried to use the examples in my webapp and it fails to transform my element.
Here is a quick demo: https://codesandbox.io/s/musing-maxwell-qo549?file=/src/index.js

It logs the correct values, but the element is not moving. Do I have to add the styling `transform: translate(x, y)` somewhere? I'm coming from Vanilla JS.

Thanks!

Edit: Should've posted this in react-use-gesture.

**Comment by Gi972** [comment, marked as answer]
Hello,

I think you use the v9 style Api 
[doc react-gesture](https://use-gesture.netlify.app/docs/)  ---> Using React Spring v8

 ;-)

**Comment by j-o-o** [comment]
YES! Thank you, I knew it was something minor.
